DevelopmentTools/HCALLMapDumper.cc

```diff
@@ -1197,20 +1201,21 @@ void HCALLMapDumper::makedbHOCalibLMapObject(std::string HCALLMapDbStr, std::str
 
   //Create Table in SQL
   //i(Unique key)
-  //Side Eta Phi dPhi Depth Det
+  //Side Eta Phi dPhi ch_type Name
```

Comment:
Please do not change the name from depth to ch_type. This will break the QIE table generation code.